app/src/app/api/keys/__tests__/route.test.ts

```
@@ -0,0 +1,279 @@
+    let whereCalled = false;
+    const chainWithSpy = {
+      from: () => chainWithSpy,
+      where: (...args: unknown[]) => {
+        whereCalled = true;
+        // Drizzle passes an SQL expression — verify it was called at all
+        expect(args.length).toBeGreaterThan(0);
+        return Promise.resolve([]);
+      },
+      innerJoin: () => chainWithSpy,
+      leftJoin: () => chainWithSpy,
+      then: (fn: (v: unknown[]) => void) => Promise.resolve([]).then(fn),
```

Biome (2.4.6)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)